homeassistant/components/reolink/update.py

```diff
@@ -64,7 +63,10 @@ def latest_version(self) -> str | None:
         if not self.coordinator.data:
             return self.installed_version
 
-        return self.coordinator.data
+        if isinstance(self.coordinator.data, str):
+            return self.coordinator.data
+
+        self.coordinator.data.version_string
```

Comment:
This returns nothing intentionally?

Reply:
whoops, good spot, thank you.
Added the missing return now.